dynatrace-oneagent-operator/README.md

```diff
@@ -39,6 +39,27 @@ Generate an API and a PaaS token in your Dynatrace environment.
 
 https://www.dynatrace.com/support/help/reference/dynatrace-concepts/why-do-i-need-an-environment-id/#create-user-generated-access-tokens
 
+#### Apply CustomResourceDefinition
+
+Download the latest version of the CRD attached to [the latest release](https://github.com/Dynatrace/dynatrace-operator/releases) and apply it.
```

Comment:
Shouldn't this link to the [dynatrace-oneagent-operator](https://github.com/Dynatrace/dynatrace-oneagent-operator/releases)?

Reply:
Updated the links and the commands accordingly